### ISSUE TITLE
minishift setup should get the compatible version of docker machine xhyve plugin. 

### ISSUE THREAD

**Issue by LalatenduMohanty** [minishift/minishift#3078]
Followup issue for https://github.com/minishift/minishift/pull/3067#discussion_r241317767

**Comment by anjannath** [comment]
Before we can implement code to fetch the compatible version of `docker-machine-driver-xhyve` we need to release (in mahcine-drivers org) the binaries for all the versions of macOS we support.

**Comment by gbraad** [comment]
Can you explain which binaries we need to provide (driver?)? And where in the organisation? I do not remember we had a xhyve/hyperkit repo. Isn't there an alnternative by utilizing `brew` ?

**Comment by anjannath** [comment]
@gbraad Right the `docker-machine-driver-xhyve` binaries, and we need to provide them here: https://github.com/machine-drivers/docker-machine-driver-xhyve/releases
> Isn't there an alnternative by utilizing brew ?

well, then we need brew installed in the host system, if we can avoid that dependency it'd  be nicer, don't you think?

**Comment by LalatenduMohanty** [comment]
@anjannath @praveenkumar We have always discussed about how the xhyve docker machine driver plugin might be incompatible to various OSX versions and we need to compile it for the particular version. However I do not remember a single instance of issue originated from incompatibility or may be I missed it. Can you you point me to one or if have you reproduced the issue locally.

**Comment by anjannath** [comment]
@LalatenduMohanty I haven't found issues till now with it, i was able to start minishift using the currently available binaries on macOS mojave, which i am certain the binaries weren't compiled on. But i am only an occasional user of macOS, i don't use it everyday so....

**Comment by gbraad** [comment]
I can remind myself two situations, but they were caused due to upgrades to Sierra or High Sierra, while the user was still using the older binaries.

**Comment by praveenkumar** [comment]
@LalatenduMohanty it depends, if you used brew to install it then it will compile it first as per your current used version of Mac and then all the updates also do same so you might not seen it.

**Comment by LalatenduMohanty** [comment]
> @LalatenduMohanty it depends, if you used brew to install it then it will compile it first as per your current used version of Mac and then all the updates also do same so you might not seen it.

I do not remember how I installed it, but can do a test around it.
